src/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -38,7 +39,7 @@ protected void launchInNewThread(final Component parent) {
     ServerGame game = null;
     try {
       m_gameData.doPreGameStartDataModifications(m_playerListing);
-      final IServerMessenger messenger = new DummyMessenger();
+      final IServerMessenger messenger = mock(IServerMessenger.class);
```

Comment:
Is this code covered by any tests? Is it only called from test code? Alternatively we need to do manual testing, which is expensive and quickly no longer a solution, or we add unit tests (or remove code if something is *only* called by test)